julia_sets/main.js

```diff
@@ -1,9 +1,19 @@
-// Reference: https://en.wikipedia.org/wiki/Julia_set
-const generateColor = () => {
-    const randomColor = Math.floor(Math.random() * 16777215).toString(16);
-    return `#${randomColor}`
+const generateColor = (iter) => {
```

Comment:
Please update the location where `generateColor` is called and pass the `iter` object. If it is not passed, we'll get `NaN`s.

Also, it'd be great if you could add the resource you mentioned in the PR description as a comment.

Reply:
Of course, sorry, I did it locally but it seems I forgot to update that part too (I didn't want to just paste everything because I made some other unnecessary changes elsewhere in the file).

Reply:
Thank you! I merged it!